pkg/config/config.go

```diff
@@ -228,9 +228,10 @@ type Config struct {
 		} `mapstructure:"ui_config"`
 	} `mapstructure:"auth"`
 	Blockstore struct {
-		Type                   string  `mapstructure:"type" validate:"required"`
-		DefaultNamespacePrefix *string `mapstructure:"default_namespace_prefix"`
-		Local                  *struct {
+		Type                        string  `mapstructure:"type" validate:"required"`
+		DefaultNamespacePrefix      *string `mapstructure:"default_namespace_prefix"`
+		EnsureRootNamespaceRWAccess bool    `mapstructure:"ensure_root_namespace_rw_access"`
```

Comment:
Yay, naming!  AFAICT this only causes read-access to be verified, and not even that (if you have just new repos).

Consider
```suggestion
		ReadableRootNamespace bool    `mapstructure:"readable_root_namespace"`
```

Also arguably this is better a part of Graveler than of Blockstore: it has no influence on blockstore behaviour.

Reply:
@nopcoder probably can say.